DarkmiraTour/community-event-manager
# [#87] Dashboard

# PR information

| Q             | A
| ------------- | ---
| Branch?       | Develop
| Bug fix?      | no
| New feature?  | yes
| Tests pass?   | yes
| Related issue | #87 

# Description

![Capture d’écran - 2019-03-29 à 15 21 34](https://user-images.githubusercontent.com/1588144/55253523-dcb41d00-5255-11e9-8fa5-46a8ab281f1b.png)

- Add Dashboard with widgets:
    - schedule days
    - sponsorships level
    - PDF brochure
- Add sidebar and move menu header into sidebar
- Update FontAwesome from 4.7 to 5 to use newer free icons

## To test it

- Don't forget to update npm dependencies and rebuild assets:
``` bash
$ docker-compose exec node bash
    $ npm install
    $ npm run watch
```
- Log in as admin
- Go to `/`

You should see widgets and sidebar.
